libs/api/domains/national-registry-x-road/src/lib/nationalRegistryXRoad.resolver.ts

```diff
@@ -95,4 +104,19 @@ export class NationalRegistryXRoadResolver {
   ): Promise<NationalRegistryCitizenship | null> {
     return this.nationalRegistryXRoadService.getCitizenship(person.nationalId)
   }
+
+  @Query(() => NationalRegistryXRoadChildGuardianship, {
+    name: 'nationalRegistryUserV2ChildGuardianship',
+    nullable: true,
+  })
+  @Audit()
+  async resolveNationalRegistryChildGuardianship(
```

Comment:
As this is a query not a field resolver you should remove the `resolve` prefix (and I would also shorten it and remove `nationalRegistry`) and move it up above the field resolvers (`@ResolveField` decorated methods).
```suggestion
  async childGuardianship(
```